tests/test_xtalimaging.py

```diff
@@ -16,7 +18,8 @@ def test_xtal_imaging(testdb):
     assert sid is not None
     assert sid > 0
 
-    si_full_path = "/dls/i03/data/2018/cm14451-99/something_new.jpg"
+    secs = int(datetime.datetime.now().timestamp())
```

Comment:
Simpler to use `time.time()`?
```suggestion
    secs = int(time.time())
```